### PR TITLE
NEGRAH GCP fixes #117 

### LINKED ISSUE #117: Improving Indentation and adding Comments in codebase
## Problem
There are few indentation issues and the code lacks the comments that would help it be easier to read.

## Description
Adding suitable comments in the codebase and maintaining-proper indentation throughout

## Additional context
Comment serve as a reminder to change something in the future. Code commenting also becomes invaluable when you want to ‘comment out’ code. This is when you turn a block of code into a comment because you don’t want that code to be run, but you still want to keep it on hand in case you want to re-add it.

## Implementation
- [x] I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by NEGRAH-S**
## Description

I added few comments from my end that will elaborate more to its understanding as well as I made some changes to indentation.

## Changes Made
-Added Comments
-Indentation was done

## Checklist
- [x] I have read and followed the contributing guidelines.
- [x] I have tested my changes thoroughly and they work as expected.
- [x] I have added necessary tests for the changes made.
- [x] I have updated the documentation to reflect the changes made.
- [x] My code follows the project's coding style and standards.
- [x] I have added appropriate commit messages and comments for my changes.

## Related Issues
issue
- #117 

## Additional Notes
This was done with lots of hardwork and diligences any update or ammendment will be highly respected.

### CODE REVIEW

**Comment by mshudrak** [comment]
Please accept Google CLA before I can review this commit. Adjust commit message to better summarize what you want to change in this PR.

**Comment by Bhardwaj-Himanshu** [comment]
Hi @NEGRAH-S, if you are still unable to figure out why Google-CLA is stopping you from making a change.

Ensure both of your Github-account- @NEGRAH-S and your Local-Git-Machine account-<ne***h​@Negrahs-MacBook-Pro.local> are linked with a single Email account.

Also all the commits should be made after first [switching your github email to public](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-email-preferences/setting-your-commit-email-address) and then use ```git config --global user.email "MY_NAME@example.com"``` to configure your local repoistory's Email Address.

I hope this helps, cause I just figured it out recently too!
Thanks

File: scanner.py [hunk 1]
```diff
@@ -1,18 +1,19 @@
-#!/usr/bin/env python3
+'''!/usr/bin/env python3
```
Comment (ZetaTwo) [inline scanner.py:1]: This whole block is incorrect. Please revert

File: src/gcp_scanner/__init__.py [hunk 2]
```diff
@@ -0,0 +1 @@
+#It is a empty file as of now.
```
Comment (ZetaTwo) [inline src/gcp_scanner/__init__.py:1]: Remove comment

File: src/gcp_scanner/__main__.py [hunk 3]
```diff
@@ -15,8 +15,8 @@
 """The main module of GCP Scanner.
 
 """
-
+# imprted main module of GCP
```
Comment (ZetaTwo) [inline src/gcp_scanner/__main__.py:18]: Redundant comment

File: src/gcp_scanner/__main__.py [hunk 4]
```diff
@@ -15,8 +15,8 @@
...
 from . import scanner
 
-if __name__ == '__main__':
-  scanner.main()
+if __name__ == "__main__":
```
Comment (ZetaTwo) [inline src/gcp_scanner/__main__.py:21]: Why " instead of '?

File: src/gcp_scanner/arguments.py [hunk 5]
```diff
@@ -13,120 +13,135 @@
...
 
-"""Command-Line Argument Parser
-
-"""
+# Command-Line Argument Parser
 
 import argparse
 import logging
 
+# Defined a function using imported module
```
Comment (ZetaTwo) [inline src/gcp_scanner/arguments.py:21]: Redundant comment

**Review by ZetaTwo — CHANGES_REQUESTED** [review]
It's very difficult to follow what changes are where and why. Please squash the commits and separate the whitespace changes from the comment changes so that I can continue the review.

**Comment by NEGRAH-S** [comment]
> Please accept Google CLA before I can review this commit. Adjust commit message to better summarize what you want to change in this PR.

mam i have accepten it but don't know why it is showing like that let me check the linked email for both

**Comment by Bhardwaj-Himanshu** [comment]
Hi @NEGRAH-S, Hope you are doing well mate!
If you are still facing the issue of not being able to fix the "Verification by Google-CLA" issue, I would be happy to help you over that.

**Comment by NEGRAH-S** [comment]
@Bhardwaj-Himanshu  hey bro, yeah I would highly admire if you could help in same.

**Comment by Bhardwaj-Himanshu** [comment]
@NEGRAH-S, I am very happy to help brother!
But first I would be require some help from your side.
Could you please share your github email id: the one you use to sign in.
and the share me the information which appears in terminal after typing `git config --list`

Thanks

**Comment by NEGRAH-S** [comment]
@Bhardwaj-Himanshu  
My GitHub email- negrahsingh32@gmail.com 
other credential as per my terminal :
Screenshot 2023-04-11 at 7 00 46 PM

**Comment by Bhardwaj-Himanshu** [comment]
Hi @NEGRAH-S,
Thanks for the images,
Go to github settings now-->emails-->"set my email address to private"(below primary email address options)--> set this either to public.

OR

You can change your "GIT-CONFIG USEREMAIL <to your private id as shown on github>"

This link is a detailed guide, please do check it out-https://help.umbler.com/hc/en-us/articles/360007180671-Turning-your-e-mail-public-on-GitHub

**Comment by NEGRAH-S** [comment]
@Bhardwaj-Himanshu  I did as u said then also updated my CLA details it shows updated but still issue is not resolved. can we connect on some  private medium for solve the issue. 
Screenshot 2023-04-12 at 7 12 40 PM

**Comment by Bhardwaj-Himanshu** [comment]
@NEGRAH-S, Thanks for the feedback.

Now that you have updated your email both on your local machine and github--> I want you to check if your email is marked **private or public** on github, using the link- https://help.umbler.com/hc/en-us/articles/360007180671-Turning-your-e-mail-public-on-GitHub

Coming on to connecting privately, I would love to do so, so here is my linkedin-https://www.linkedin.com/in/himanshu--bhardwaj/

But let's keep this discussion open and public here on github, so that if I am unable to help, others can look into it and help you.

After opting for public email id on github, I want you to again commit the changes first by making a small change on your local machine, and then upload them into github into another branch or `trying using same branch even #117` for your forked repository.

Then let the checks take place and let me know via a screenshot what happens!

**Comment by NEGRAH-S** [comment]
As discussed with @Bhardwaj-Himanshu  I have decided to close this PR and make new with changes mentioned by @ZetaTwo .

**Comment by Bhardwaj-Himanshu** [comment]
@mshudrak, He'll open this pull request again, please don't close this until now.
@NEGRAH-S, do link this issue, while create a new PR.

**Comment by NEGRAH-S** [comment]
> @mshudrak, He'll open this pull request again, please don't close this until now.
> @NEGRAH-S, do link this issue, while create a new PR.

okh bro I got that will link the issue to my new PR this time.

**Comment by Bhardwaj-Himanshu** [comment]
Also, before changing the lines could you please add `"editor.formatOnSaveMode": "modifications",`--this line of code to your settings.json file?
What would it do?
-prevent unnecessary introduction of white spaces and blank lines and would only show actual lines modified while doing a commit.
-To do this do **Ctrl+Shift+P in vscode --> settings.json in the commmand palette which appears--> open it--> add `"editor.formatOnSaveMode": "modifications"`--> save**

OR you could watch this youtube video for the same, https://www.youtube.com/watch?v=-7t0ewz7-jk

File: src/gcp_scanner/arguments.py [hunk 6]
```diff
@@ -13,120 +13,135 @@
...
+        help="Set logging level (INFO, WARNING, ERROR)",
+    )
+    parser.add_argument(
+        "-lf",
+        "--log-file",
+        default=None,
+        dest="log_file",
+        help="Save logs to the path specified rather than displaying in\
+ console",
+    )
```
Comment (Bhardwaj-Himanshu) [inline src/gcp_scanner/arguments.py:130]: Could you also try removing the unnecessary whitespaces and indentations, these create a great amount of confusion when reviewing the file and some changes could go unnoticed.

Thanks!

**Comment by Bhardwaj-Himanshu** [comment]
@NEGRAH-S Do update this issue with your progress.

Or else @mshudrak @ZetaTwo please close this issue as it's unattended!

**Comment by NEGRAH-S** [comment]
> @NEGRAH-S Do update this issue with your progress.
> 
> Or else @mshudrak @ZetaTwo please close this issue as it's unattended!

hello @Bhardwaj-Himanshu  , I was occupied with some work but I saw someone has made an PR addressing same issue , so I feel this PR can be closed .

**Pull request closed without merging**